scripts/create-s3-buckets.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+# Executable that waits for a port at an address to be open.
+# For further details go to: https://github.com/ufoscout/docker-compose-wait
+/wait
+
+awslocal s3 mb s3://unstable-elife-style-content-adapter-incoming
```

Comment:
`unstable` is the name of an environment where this is deployed; the convention for the environment that is used locally is `dev`, or more in general `ci` if it is running in Travis CI or a similar place. I would go for `dev-elife-style-content-adapter-incoming`, similarly for the other bucket?